test/runtests.jl

```diff
@@ -15,34 +16,32 @@ function test_weights()
     return [reshape(Float32.(1:prod(s)), s) for s in shapes]
 end
 
-# This simple model should work with both Flux's `params/loadparams!` and
-# our `weights/load_weights!`. The only difference is in layers with `!isempty(other_weights(layer))`.
-@testset "using ($get_weights, $load_weights)" for (get_weights, load_weights) in [(fetch_weights, load_weights!, params, Flux.loadparams!)]
```

Comment:
This isn't really necessary - it's just checking that `params` and `Flux.loadparams!` work to (de)-serialize the model (as well as LegolasFlux), but that doesn't really matter for us.